src/lib/components/workflow/workflows-summary-table-with-filters.svelte

```diff
@@ -140,35 +141,28 @@
             {/if}
           </div>
         </th>
-        <th class="table-cell md:w-auto" />
-        <th class="table-cell xl:w-60" />
-        <th class="hidden xl:table-cell xl:w-60" />
-        <th class="hidden xl:table-cell xl:w-60" />
+        {#each Array(columns.length - 1) as _, index}
```

Comment:
I've opted for this hack rather than updating the `colspan` based on a breakpoint since that would require either creating a reusable `BulkActions` component or duplicating code here.

We should probably rethink this once we decide what responsiveness will look like on the new workflows table.